qiime/metadata.py

```diff
@@ -14,12 +16,30 @@ def __init__(self, dataframe):
     def __repr__(self):
         return self._dataframe.to_json()
 
+    @staticmethod
+    def load(path):
+        try:
+            df = pd.read_csv(path, sep='\t', index_col=0, dtype=object)
```

Comment:
Confirmed that that's an issue, and fixed (note that it's not a bug in pandas though, ``dtype`` here is specifically for the columns, not the index).